### DISCUSSION TITLE
No Fail score multiplier only applies if you would have failed a map

### DISCUSSION THREAD

**Discussion by kissinglucifer** [ppy/osu#21865] [Ideas] [answered]
As it says in the title, The no Fail multiplier of x.50 would only come into effect if you would have failed the map. And as a customization option it would have an option to set the multiplier to .50. 

Beat saber implemented this a few years back and it made the game a lot more fun. Playing a tech map without having to restart would be **a lot** more fun

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/18424
https://github.com/ppy/osu/discussions/16779
https://github.com/ppy/osu/discussions/14114
https://github.com/ppy/osu/issues/9844
